divvy/cloudfilebrowser.js

```diff
@@ -213,15 +220,20 @@ var provision = (function() {
             if(elementDetails != null && elementDetails != undefined) {
 
                 // if onedrivebusiness
-                if (element === 'onedrivebusiness') {
+                if (element === 'onedrivebusiness' || element === 'sharepoint') {
                     var siteAddress = cbArgs.siteAddress;
                     if (!!siteAddress) {
                         elementDetails.siteAddress = siteAddress;
                     } else {
                         return;
                     }
                 }
-                
+
+                if (element === 'sharepoint') {
```

Comment:
We can improve this a little by moving this if-statement inside of the one at Line 223, that way it _only_ runs if the element is ODFB or SharePoint.